setup.cfg

```diff
@@ -0,0 +1,11 @@
+[metadata]
+name = usmqe
+
+[options]
+zip_safe = False
+include_package_data = True
+packages = find:
+entry_points = file:entry_points.txt
+
+[options.packages.find]
+exclude = usmqe.usmqe_tests
```

Comment:
There seems to be missing "new line" at the end of this file... (I'm not sure, if it is intended or if it could cause any issue... consider it just as note..)

Reply:
Noted. I'm going to add a newline along with other minor updates (in readme files) after this is merged.